### PR TITLE
Align input options for --include and --subset arguments

### PR DESCRIPTION
**Pull request by djptek**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)? Y
- Have you followed the [contributor guidelines](https://github.com/elastic/ecs/blob/master/CONTRIBUTING.md)? Y
- For proposing substantial changes or additions to the schema, have you reviewed the [RFC process](https://github.com/elastic/ecs/blob/master/rfcs/README.md)? Y
- If submitting code/script changes, have you verified all tests pass locally using `make test`? Y 
- If submitting schema/fields updates, have you generated new artifacts by running `make` and committed those changes? Y
- Is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed. Y 
- Have you added an entry to the [CHANGELOG.next.md](https://github.com/elastic/ecs/blob/master/CHANGELOG.next.md)? Y

Addresses https://github.com/elastic/ecs/issues/899

### CODE REVIEW

**Comment by djptek** [comment]
In Draft as need to add tests

**Comment by djptek** [comment]
low level tests passing, working with include, exclude, subset. next step add higher level tests

File: CHANGELOG.next.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ Thanks, you're awesome :-) -->
 
 ### Schema Changes
 
+* Align input options for --include and --subset arguments #1519
```
Comment (kgeller) [inline CHANGELOG.next.md:13]: This is more of a tooling addition than a schema change, right?

**Review by kgeller — APPROVED** [review]
Looks great Dominic!

File: CHANGELOG.next.md (hunk 1 above)
Comment (djptek) [reply CHANGELOG.next.md]: Indeed, thanks for spotting @kgeller

**Comment by ebeahan** [comment]
@djptek, with this change aligning the input behavior of these two arguments, can it be implemented independently from the @timestamp conversation happening in #1646?

**Comment by djptek** [comment]
@ebeahan I think I need to fix the `@timestamp` first in order otherwise this PR will permit non-conformant ECS with no `@timestamp`. I checked the tests and they are valid, it's just a question of order of execution

**Comment by ebeahan** [comment]
> I think I need to fix the @timestamp first in order otherwise this PR will permit non-conformant ECS with no @timestamp

I believe that's already true in the current tooling and not something introduced by these changes?

If this change is ready, we can introduce your improvements here and address #1646 later.

**Comment by ebeahan** [comment]
@djptek should this one be backported to `8.0`?

**Pull request merged**